packages/firestore/src/platform_node/load_protos.ts

```diff
@@ -38,5 +61,8 @@ export function loadProtos(): grpc.GrpcObject {
     root,
     file: 'google/firestore/v1/firestore.proto'
   };
-  return grpc.load(firestoreProtoFile, /*format=*/ 'proto', options);
+
+  let builder = ProtobufJS.newBuilder(options);
```

Comment:
I think you can (and probably should) add 'protobufjs' to our package.json (as a devDependency, since it's only needed for tests) with a 6.x version, and then you can use the same options as in the loadProtos() method, and we'll be sure we're loading the protos in the actual SDK and the tests in the same manner.

Reply:
Done.